packages/core/src/shader/Program.ts

```diff
@@ -45,10 +45,10 @@ export class Program
     public vertexSrc: string;
     public fragmentSrc: string;
     nameCache: any;
-    glPrograms: { [ key: number ]: GLProgram};
+    glPrograms: { [key: number]: GLProgram };
```

Comment:
Can you please revert all these whitespace changes? They are not relevant to your change and we'd rather manage whitespace with eslint.

Reply:
Ok, no problem. This was messed up by my editor. But pixi's lint-fix was fine with this whitespaces, so I decided to keep them. Maybe we should add a lint rule for them?

Reply:
Probably, but that should be another PR.